crates/eframe/src/native/wgpu_integration.rs

```diff
@@ -386,6 +412,8 @@ impl WinitApp for WgpuWinitApp {
                 log::debug!("Event::Resumed");
 
                 let running = if let Some(running) = &self.running {
+                    #[cfg(target_os = "android")]
+                    self.recreate_window(event_loop, running);
```

Comment:
In the same way we're looking to consistently emit `Suspended` before the loop is destroyed, giving users a consistent place to clean up their swapchain.

Reply:
1. On platforms other than Android, Ios and Web Resume event is only send once when loop starts running. So the recreation for other platforms won't happen and for Ios and Web the Suspend/Resume cycle is different.
2. The same for Suspend event it only occurs for Android, Ios and Web.

Reply:
On other platforms it is not about recreating, it is about _initially creating_ this state in the same place instead of having unnecessary conditionals in `eframe`.

Destruction only happens in `::Suspended`.

Reply:
Okay this looks much harder to fix than thought. `WgpuWinitRunning` contains a lot of state that should easily live outside the Android surface lifecycle.